control-plane/grpc/src/operations/pool/server.rs

```diff
@@ -37,9 +37,12 @@ impl PoolGrpc for PoolServer {
     ) -> Result<tonic::Response<pool::CreatePoolReply>, tonic::Status> {
         let req: CreatePoolRequest = request.into_inner();
         match self.service.create(&req, None).await {
-            Ok(pool) => Ok(Response::new(CreatePoolReply {
-                reply: Some(create_pool_reply::Reply::Pool(pool.into())),
-            })),
+            Ok(pool) => {
+                tracing::event!(target: "target-mbus", tracing::Level::INFO, event_data = &pool.event().to_string());
```

Comment:
nit: Could this be put inside a macro or a function (which accepts a dyn ToString), because of how often this is used?